Core/Util/GameManager.cpp

```diff
@@ -131,8 +151,10 @@ bool GameManager::Uninstall(std::string name) {
 void GameManager::Update() {
 	if (curDownload_.get() && curDownload_->Done()) {
 		INFO_LOG(HLE, "Download completed! Status = %d", curDownload_->ResultCode());
-		Path fileName = Path(curDownload_->outfile());
+		Path fileName = curDownload_->outfile();
 		if (curDownload_->ResultCode() == 200) {
+			// TODO: This fails. Wonder if there's a race condition?
```

Comment:
What does it get for result code?  I thought download was working fine not long ago.

-[Unknown]

Reply:
This is outdated.